kotest-property/src/commonMain/kotlin/io/kotest/property/arbitrary/collections.kt

```diff
@@ -11,6 +14,12 @@ import kotlin.random.nextInt
  */
 fun <T> Arb.Companion.element(collection: Collection<T>): Arb<T> = arb { collection.random(it.random) }
 
+/**
+ * Returns an [Arb] whose values are chosen randomly from those in the supplied collection.
+ * May not cover all items. If you want an exhaustive selection from the list, see [Exhaustive.collection]
+ */
+fun <T> Arb.Companion.element(vararg collection: T): Arb<T> = arb { collection.random(it.random) }
```

Comment:
This is an unrelated change, but I thought it would be a useful overload.

Reply:
Assuming this overload is acceptable, it does seem like this is redundant with `constant`. The result of `Arb.element(a)` should be functionally equivalent to `Arb.constant(a)`. Would it be reasonable to remove `constant`?

Reply:
At least make one a typealias. I think constant is handy for discoverability.

Reply:
OK, I changed constant to call element.